src/compas/files/xml_cli.py

```diff
@@ -146,11 +193,11 @@ def _parse_xml_declaration(self, xml_decl):
     def _start_element(self, reader):
         """Notify the tree builder that a start element has been
         encountered."""
-        name = reader.LocalName
+        name = reader.Name
```

Comment:
I'm terribly confused about this.  When I run the code from the tests in rhino on my machine, it works.  I have no idea why all the ironpython tests are failing now.

Reply:
I replaced all references to `LocalName` to `Name` and all tests pass.  I looked briefly at the source code of `aglyph.compat.ipyetree` and did not see a good reason why `LocalName` is used there.  I've included two (nearly identical) tests involving namespaces.  One for xml and one for urdf.  But probably there should be more.  Ideas?